src/test/index.js

```diff
@@ -80,6 +80,5 @@ describe( 'Gutenberg Mobile initialization', () => {
 		const blockList = screen.getAllByTestId( 'block-list-wrapper' )[ 0 ];
 
 		expect( blockList ).toBeVisible();
-		expect( console ).toHaveLoggedWith( 'Hermes is: true' );
```

Comment:
Hey @dcalhoun 👋 does this change look correct? I was having CI failures and saw that this https://github.com/WordPress/gutenberg/pull/53268 change was recently merged. It looks like we shouldn't assert that this was logged anymore. Is it accurate? Let me know, thank you!

Reply:
Yes, you are correct. This assertion needs removal. Thank you for taking care of this.